test/ueberauth/strategy/cognito_test.exs

```diff
@@ -218,11 +267,12 @@ defmodule Ueberauth.Strategy.CognitoTest do
         |> Plug.Conn.fetch_query_params()
         |> Cognito.handle_callback!()
 
-      assert conn.private.cognito_id_token == %{"email" => "foo"}
+      _id_token = conn.private.cognito_id_token
+      assert match?(_id_token, %{"email" => "foo"})
```

Comment:
This is because the warning `warning: variable "id_token" is unused (if the variable is not meant to be used, prefix it with an underscore)` is displayed.

However, it worked well with `assert %{"email" => "foo"} = conn.private.cognito_id_token`, so I'll change it.

Thank you!

Reply:
That sounds like a bug in Elixir: you should file a bug with them. Always an interesting day to find a bug in the language itself!